packages/block-library/src/page-list-item/edit.js

```diff
@@ -50,6 +55,12 @@ export default function PageListItemEdit( { context, attributes } ) {
 
 	const innerBlocksProps = useInnerBlocksProps( blockProps );
 
+	useEffect( () => {
+		if ( isSelected ) {
+			setAttributes( { isSelected: true } );
```

Comment:
Is there a way to achieve this without setting an attribute?

Reply:
not sure if onChange of `innerBlocks` is triggered by anything else. There is also an `onInput` 👁️